Initium-Odp/src/com/universeprojects/miniup/server/services/MovementService.java

```diff
@@ -1,65 +1,79 @@
-package com.universeprojects.miniup.server.services;
```

Comment:
I'm not sure why Github can't handle this one in the pull request. It's not needed. This was the file that when we first discussed this you mentioned adding the isPathDiscovered. I ended up not needing it after spfiredrake mentioned a no-query way to solve the issue. I had asked and you said to just leave it. I can re-do everything again and just not include it. It's not being used. I just left it because you said it was okay to leave it. I'll do that tonight when I get back home.